ENGDOCS-1733 (#18457)
docker/docs: content/desktop/hardened-desktop/settings-management/_index.md

### Who is it for?

> - For organizations who wish to configure Docker Desktop to be within their organization's centralized control.
> - For organizations who want to create a standardized Docker Desktop environment at scale.

Reviewer: WDYT?

```suggestion
- For organizations that want to configure Docker Desktop to be within their organization's centralized control.
- For organizations that want to create a standardized Docker Desktop environment at scale.
```
